### ISSUE TITLE
Clickable Tab Title

### ISSUE THREAD

**Issue by jcuenod** [ticlo/rc-dock#69]
Is it possible to respond to clicks on tabs (as opposed to mousedowns that initiate drag)? Something like:

```jsx
let mainTab = {
	content: <SomeComponent />,
	title: <a href="#" onClick={() => console.log("do something!")}>Clickable Title</a>,
	id: 'tab1'
}
```

**Comment by flieks** [comment]
I am also interested in this for changing the tab title.

I tried the onClick on a <div> in the title but it does not trigger. Understandable because its needed for the dragdrop.

Maybe we can have a ondblclick working ? Tested but does not trigger.

**Comment by rinick** [comment]
right now it's not clickable, I will see if I can fix that

**Comment by rinick** [comment]
resolved in 3.0.5

**Comment by flieks** [comment]
Thanks alot. I can confirm this is working. I can edit my title using an inline input field (title= Component)

**Comment by jcuenod** [comment]
Haven't tested this yet but just want to say, @rinick, thank you so much for prompt support! I know maintaining OSS projects can be thankless and tiring and then coupled with expectations from people who use your code. So thank you for your help on the issues I've filed recently!

**Comment by flieks** [comment]
@rinick Can it be that this does not work anymore in the latest version ?

**Comment by rinick** [comment]
> @rinick Can it be that this does not work anymore in the latest version ?

I will check that tomorrow

**Comment by rinick** [comment]
@flieks 
fixed in 3.0.16
